torch/csrc/jit/runtime/register_cuda_ops.cpp

```diff
@@ -18,19 +18,47 @@ c10::AliasAnalysisKind aliasAnalysisFromSchema() {
 
 RegisterOperators const reg({
     Operator(
-        "cuda::current_stream(int64_t val) -> __torch__.torch.classes.cuda.Stream",
+        "cuda::current_stream.device(Device? device) -> __torch__.torch.classes.cuda.Stream",
         [](Stack* stack) {
-          auto idx = uint16_t(pop(stack).toInt());
-          auto s = c10::cuda::getCurrentCUDAStream(idx);
+          auto device = pop(stack).toOptional<c10::Device>();
+          c10::DeviceIndex device_index = device.has_value()
+              ? device->index()
+              : c10::cuda::current_device();
```

Comment:
Why is this `c10::cuda::current_device()` here but `-1` below?

Reply:
This is due to clang-formatting.

Reply:
`clang-format` changed the content of the code?

Reply:
No, sorry. I misread your question. No particular reason. `c10::cuda::current_device()`  returns the current device index. If we pass -1 to `getCurrentCUDAStream`, it uses the the current device to get the `stream`. Let me make it more consistent.

Reply:
Fixed now.